acceptance/tests/fixtures/cases/static-client-openshift-inject/patch.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-client
+spec:
+  template:
+    metadata:
+      annotations:
+        "consul.hashicorp.com/connect-inject": "true"
+        "consul.hashicorp.com/connect-service-upstreams": "static-server:1234"
+        "k8s.v1.cni.cncf.io/networks": '[{ "name":"consul-cni" }]'
```

Comment:
I think at some point we've added some openshift specific SCCs to static-server or static-client base cases (or both) and with this change, would it make sense to clean those up or do we still need them?

Reply:
The privileged SCC is needed for when you're not using CNI and so your init container needs elevated privileges to run iptables